cli/command/stack/list_test.go

```diff
@@ -48,7 +48,7 @@ func TestListErrors(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		cmd := newListCommand(test.NewFakeCli(&fakeClient{
+		cmd := newListCommand(test.NewFakeCliWithOutput(&fakeClient{
```

Comment:
Could just be `NewFakeCli`, but I know you're not trying to exhaustively convert these.